wandb/sdk/wandb_settings.py

```diff
@@ -656,6 +657,10 @@ def _default_props(self) -> Dict[str, Dict[str, Any]]:
                 ),
                 "auto_hook": True,
             },
+            _file_stream_timeout_seconds={
+                "value": 30,
```

Comment:
30s feels risky, let's bump to to at least 60.

```suggestion
                "value": 60,
```